src/ImageProcessorCore/Colors/ColorTransforms.cs

```diff
@@ -18,9 +19,60 @@ namespace ImageProcessorCore
     public partial struct Color
     {
         /// <summary>
+        /// Blends two colors by lightening.
+        /// </summary>
+        /// <remarks>
+        /// Selects the lighter of the backdrop and source colors.
+        /// The backdrop is replaced with the source where the backdrop is lighter; otherwise, it is left unchanged
+        /// Note: This behavior is on a channel by channel basis, i.e., this rule is applied to each of the 3 RGB color channels separately.
+        /// </remarks>
+        /// <param name="backdrop"></param>
+        /// <param name="source"></param>
+        /// <returns>
+        /// The <see cref="Color"/>.
+        /// </returns>
+        public static Color Lighten(Color backdrop, Color source)
+        {
+            var newVector = new Vector4
```

Comment:
The `Cr = (1 - αb) x Cs + αb x B(Cb, Cs)` works for opaque pre-multpilied because you're multiplying the blending function by 1! :+1: 

Now we have to determine if we're intending the ColorExtensions to be ImageProcessor internal or to be used by clients too, because we'll need to differentiate a "post multiplied blend" with a "pre multiplied blend" in the API for client use. What do you think?